documents-webapp/src/main/webapp/vue-app/documents/components/DocumentsMain.vue

```diff
@@ -515,10 +518,7 @@ export default {
           nodePath = nodePath.substring(0,nodePath.lastIndexOf('/'));
         }
         this.getFolderPath(nodePath);
-      } else {
-        this.parentFolderId=null;
-        this.fileName=null;
-        this.folderPath='';
+      } else if (this.selectedView === 'folder') {
         this.selectedView = 'timeline';
```

Comment:
```suggestion
```
Shouldn't switch to timeline here IMO

Reply:
I think the same as you. so I delete this line.